plugin/storage/kafka/options_test.go

```diff
@@ -28,12 +28,18 @@ func TestOptionsWithFlags(t *testing.T) {
 	command.ParseFlags([]string{
 		"--kafka.producer.topic=topic1",
 		"--kafka.producer.brokers=127.0.0.1:9092, 0.0.0:1234",
-		"--kafka.producer.encoding=protobuf"})
+		"--kafka.producer.encoding=protobuf",
+		"--kafka.producer.required.acks=1",
+		"--kafka.producer.compression=0",
+		"--kafka.producer.compression.level=-1000"})
```

Comment:
Should use non-default values for these properties, to ensure they are being set.